tools/upload_tables.sh

```diff
@@ -198,6 +201,15 @@ convert_and_insert_values() {
 		return 1
 	fi
 
+	if  [[ -n "${GCP_INSTANCES:-}" ]]; then
```

Comment:
Please simplify this piece of code be removing `grep` and checking for null `src_addr` after the if/else block.  You also don't need ":-" in the variable expansion of GCP_INSTANCES.